modules/administration-guide/pages/collecting-logs-using-chectl.adoc

```diff
@@ -5,11 +5,38 @@
 [id="collecting-logs-using-{prod-cli}_{context}"]
 = Collecting logs using {prod-cli}
 
+An installation of {prod} consists of several containers running in the {orch-name} cluster. While it is possible to manually collect logs from each running container, `{prod-cli}` provides commands which automate the process.
+
 Following commands are available to collect {prod} logs from the {orch-name} cluster using the `{prod-cli}` tool:
 
+`{prod-cli} server:logs`:: 
+Collects existing {prod} server logs and stores them in a directory on the local machine. By default, logs are downloaded to a temporary directory on the machine. However, this can be overwritten by specifying the `-d` parameter. For example, to download Che logs to the `/home/user/che-logs/` directory, use the command
```

Comment:
No, with `chectl server:logs`, the `chectl` tool collects logs from the various containers that comprise an Eclipse Che installation and saves them to the local disk (i.e. your laptop).

Reply:
How about this then?

```suggestion
The 'chectl' tool collects existing {prod} server logs and stores them in a directory on the local machine. By default, logs are downloaded to a temporary directory on the machine. However, this can be overwritten by specifying the `-d` parameter. For example, to download Che logs to the `/home/user/che-logs/` directory, use the command
```

Reply:
I'm happy to change it, but I'm not sure why it's necessary -- the current format of this section is
```
The following commands are available:
* <command>
    * <what it does>
```
e.g. 
![che-doc-logs](https://user-images.githubusercontent.com/16168279/162041115-211a49de-de05-4095-bc72-a00b5516f8cf.png)

In this case I think it should be clear that `chectl` is what's doing the log collecting, since its the command you're running.

Reply:
agree + we should not use the `chectl` in general, it should be externalized as `{prod-cli}`
@Eva-Lotte can we merge in the current state? if yes, please approve